### PR TITLE
DOCSP-31519 Query Explain Plan UI Change

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION
- Updates information on query explain plans for Compass 
- Applied task page info typing

## STAGING
https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-31519-explain-button/query-plan/#steps

## JIRA
https://jira.mongodb.org/browse/DOCSP-31519

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64c9115987b3b36046e6b2d1
## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/query-plan.txt [hunk 1]
```diff
@@ -12,47 +12,66 @@ View Query Performance
...
-  you can utilize the :manual:`.explain()
-  </reference/method/db.collection.explain>` method from :doc:`Embedded
-  MongoDB Shell </embedded-shell>` in |compass-short|.
+Steps 
+-----
+
+.. procedure:: 
+   :style: normal
+
+   .. step:: Click the Explain button
```
Comment (sarah-olson-mongodb) [inline source/query-plan.txt:48]: (minimalism)
An option throughout UI documentation is to omit references to the UI elements whenever possible. With the explain button, you could probably omit both references. 

You could also consider omit the screenshot, as I think the Explain button is pretty clear. This is very subjective though, and just a suggestion.

File: source/query-plan.txt [hunk 2]
```diff
@@ -12,47 +12,66 @@ View Query Performance
...
+         :figwidth: 696px
+         :alt: Query plan
+
+   .. step:: Select a query operation
+      
+      By default, the explain stages are are shown as a :guilabel:`Visual Tree`.
+      Each query operation appears as a node on the tree.
+
+      For more detailed execution information about the query operation, click 
+      the corresponding node.
```
Comment (sarah-olson-mongodb) [inline source/query-plan.txt:63]: (image)
I think you could go either way on this image. If you leave it in, consider adding a very brief "For example..." sentence. I think you could also omit altogether

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks so much @ajhuh-mdb. A couple comments for your consideration and LGTM!

File: source/query-plan.txt (hunk 2 above)
Comment (ajhuh-mdb) [reply source/query-plan.txt]: Updated!

**Review by gribnoysup — APPROVED** [review]
Looks awesome, thank you!

**Pull request merged**